draft-ietf-httpbis-connect-tcp.md

```diff
@@ -168,10 +168,13 @@ When closing connections, endpoints are subject to the following requirements:
 * When a TCP connection reaches the TIME-WAIT or CLOSED state, the associated endpoint MUST close its send stream.
   - If the connection closed gracefully, the endpoint MUST close the send stream gracefully.
   - Otherwise, the endpoint SHOULD close the send stream abruptly, using a mechanism appropriate to the HTTP version:
-    - HTTP/3: RESET_STREAM with H3_CONNECT_ERROR
-    - HTTP/2: RST_STREAM with CONNECT_ERROR
-    - HTTP/1.1 over TLS: a TLS Error Alert
-    - HTTP/1.1 (insecure): TCP RST.
+    - HTTP/3: RESET_STREAM with H3_CONNECT_ERROR;
```

Comment:
I think it's up to @MikeBishop, but do we want to use the terms used in RFC 9114 such as "reset a stream," rather than directly referring to the specific frames defined in RFC 9000?